gno/r/launchpad_grc20/merkleutil.gno

```diff
@@ -0,0 +1,16 @@
+package launchpad_grc20
+
+// Implementation of Hashable interface from gno/p/demo/merkle
+//
+// type Hashable interface {
+// 	Bytes() []byte
+// }
+//
```

Comment:
Why is this commented out?

Reply:
Just to show what interface i implement to be able to communicate with merkle pkg functions that use this interface as parameters, so people are not surprised that i create a struct with a Bytes method. https://github.com/gnolang/gno/blob/master/examples/gno.land/p/demo/merkle/merkle.gno